src/test/java/regions/RegionEntitiesTest.java

```diff
@@ -17,71 +17,69 @@
 package regions;
 
 import com.google.common.collect.Sets;
+import org.junit.Before;
 import org.junit.Test;
+import org.mockito.Mockito;
 import org.terasology.dynamicCities.region.RegionEntityManager;
-import org.terasology.dynamicCities.region.components.RegionEntitiesComponent;
 import org.terasology.entitySystem.entity.EntityRef;
+import org.terasology.logic.location.LocationComponent;
+import org.terasology.math.geom.Vector2i;
 import org.terasology.math.geom.Vector3f;
 import org.terasology.moduletestingenvironment.ModuleTestingEnvironment;
 
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
 import java.util.Set;
 
 import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
 
 public class RegionEntitiesTest extends ModuleTestingEnvironment {
-
-    private RegionEntitiesComponent regionEntitiesComponent;
     private RegionEntityManager regionEntityManager;
 
     private EntityRef[] test;
-    private Vector3f[] pos = new Vector3f[4];
+    private Vector3f[] pos = new Vector3f[10];
 
     @Override
     public Set<String> getDependencies() {
         return Sets.newHashSet("engine", "DynamicCities", "ModuleTestingEnvironment");
     }
 
-    /*
     @Before
-    public void setup() {
+    public void setupEntityRefs() {
         pos[0] = new Vector3f(16, 0, 16);
         pos[1] = new Vector3f(16, 0, -16);
         pos[2] = new Vector3f(-16, 0, 16);
         pos[3] = new Vector3f(-16, 0, -16);
+        pos[4] = new Vector3f(0, 0, 0);
+        pos[5] = new Vector3f(32, 0, 32);
+        pos[6] = new Vector3f(32, 0, -32);
+        pos[7] = new Vector3f(-32, 0, 32);
+        pos[8] = new Vector3f(-32, 0, -32);
+        pos[9] = new Vector3f(97, 0, -97);
 
-//        regionEntityManager = getHostContext().get(RegionEntityManager.class);
-
-//        regionEntityManager = Mockito.mock(RegionEntityManager.class);
```

Comment:
Did you check what mock does before erasing this line? I'm just curious, because it seems like it was put there for a reason... Not sure what till I'll read through it as well :P